app/templates/popover.hbs

```diff
@@ -3,5 +3,8 @@
   <h4 class="popover-title">{{title}}</h4>
 {{/if}}
 <div class="popover-content">
-  {{view view._contentViewClass}}
+  {{view _contentViewClass
+      content=content
+      isSliding=isSliding
```

Comment:
Are those parameters really needed?

Reply:
@cyril-sf I did some quick research. The aim of this change was definitely to move away from the implicit context from this template being available as the context of the `_contentViewClass`. `_contentViewClass` is often (always?) a `ContainerView`, and so the context is also inherited to the template of the `itemViewClass`.

Regardless of my original intent, I think you're correct that we shouldn't need to pass any values here for existing Iverson code to function on 1.12.

To avoid the inherited context stuff and remove use of `parentView` I expect we will need to pass `content` and maybe `popoverComponent` so we can update the content view class and item view classes.

I was in the weeds when I touched this and may be been on a wild goose chase wrt the interaction of popups and selects.